specifications/api/swagger.yaml

```diff
@@ -4124,6 +4124,17 @@ x-errMalformedTxPayload: &errMalformedTxPayload
       type: string
       enum: ['malformed_tx_payload']
 
+x-errMempoolIsFull: &errMempoolIsFull
+  <<: *responsesErr
+  title: malformed_tx_payload
+  properties:
+    message:
+      type: string
+      description: May occur when submitting a serialized transaction to an external service with full mempool.
```

Comment:
I don't think that its an external service exclusively whose mempool is full, isn't it also possible that local node fails with the same error?

Reply:
OK, but we did not have this error before .. I mean the 405 we are not producing it for the node-mode